Comp2023/src/main/java/frc/robot/subsystems/Extension.java

```diff
@@ -93,7 +93,7 @@ public class Extension extends SubsystemBase
   private boolean                         m_calibrated            = EXConsts.kExtensionCalibrated;  // Indicates whether the extension has been calibrated
 
   private Timer                           m_safetyTimer           = new Timer( ); // Safety timer for use in extension
-
+  private double                          m_extensionDistTravelled;
```

Comment:
The extension has not travelled anywhere at this point, I think it's the distance "requested" for the move. Just a naming improvement.